test/strings/basic.jl

```diff
@@ -478,8 +478,12 @@ Base.endof(x::CharStr) = endof(x.chars)
 @test cmp(CharStr("\U1f596"), "\U1f596\U1f596") == -1
 
 # repeat function
-@test repeat("xx",3) == repeat("x",6) == "xxxxxx"
-@test repeat("αα",3) == repeat("α",6) == "αααααα"
+@test repeat("xx",3) == repeat("x",6) == repeat('x',6) == "xxxxxx"
+@test repeat("αα",3) == repeat("α",6) == repeat('α',6) == "αααααα"
+@test repeat("x",1) == repeat('x',1) == "x"
+@test repeat("x",0) == repeat('x',0) == ""
+@test "x"^1 == 'x'^1 == "x"
+@test "x"^0 == 'x'^0 == ""
```

Comment:
Probably merge these two tests with the previous lines, e.g. `@test "x"^1 == 'x'^1 == repeat("x",1) == repeat('x',1) == "x"`